packages/webview/src/styles/state-edit-popup.css

```
@@ -409,3 +409,261 @@
 .state-edit-popup *::-webkit-scrollbar-thumb:hover {
   background: #94a3b8 !important;
 }
+
+/* ===== Dark Theme ===== */
+body[data-vscode-theme-kind="vscode-dark"] .state-edit-popup,
+body[data-vscode-theme-kind="vscode-high-contrast"] .state-edit-popup {
+  background: rgba(30, 30, 46, 0.95);
```

Stylelint (17.2.0)

[error] Expected "0.95" to be "95%" (alpha-value-notation)

(alpha-value-notation)